test/manual-test-examples/webgl/interaction/debugMode/sketch.js

```diff
@@ -0,0 +1,25 @@
+function setup() {
+  createCanvas(windowWidth, windowHeight, WEBGL);
+  normalMaterial();
+  let cam = createCamera();
+  cam.move(0, -this.height / 4, 0);
+  debugMode();
+}
```

Comment:
but also: I can't run this because it generates an error with validateParams in my default build...

Reply:
there's a bug in the validateParameters handling that causes a crash when the description of a {Constant}-typed parameter does not match the required format.
the fix in this case is to add the 'either' keyword before the list of valid values for the parameter ("**either** GRID, AXES, or OFF").

Reply:
indeed, going to add that now...

Reply:
sorry, i should have done this a long time ago... #3122